cobbler/remote.py

```diff
@@ -809,6 +809,15 @@ def get_item_resolved_value(self, item_uuid: str, attribute: str):
         ):
             return return_value.name
         elif isinstance(return_value, dict):
+            if len(return_value) > 0 and isinstance(
+                next(iter(return_value.values())), system.NetworkInterface
```

Comment:
Just wondering, is it correct to assume that when `system.NetworkInterface` comes as the first item in values for a dict return, then we are talking always about a structure of network interfaces?

Reply:
Yes because we asked for the `interfaces` key then. Do you desire I add that explicitly? Mixed object dicts I am not aware of to be existing atm. Or should I check all objects?

Reply:
Hmmm, I tend to prefer we explicitely check that `interfaces` was the key requested to then do this concrete conversion.